routers/api/v1/repo/file.go

```diff
@@ -136,7 +136,7 @@ func GetEditorconfig(ctx *context.APIContext) {
 	}
 
 	fileName := ctx.Params("filename")
-	def := ec.GetDefinitionForFilename(fileName)
+	def, _ := ec.GetDefinitionForFilename(fileName)
```

Comment:
```suggestion
	def, err := ec.GetDefinitionForFilename(fileName)
	if err != nil {
```

Reply:
I will remove this requested change as I found later that it conflict with what was suggested before.